micro-ui/web/micro-ui-internals/packages/modules/core/src/pages/citizen/Home/UserProfile.js

```
@@ -76,6 +87,27 @@
+        acc[key] = new RegExp(value);
+        return acc;
+      }, {})
+    );
+  };
+
+  const [validationConfig,setValidationConfig] = useState(mapConfigToRegExp(defaultValidationConfig) || {});
+
+  const { data: mdmsValidationData, isValidationConfigLoading } = Digit.Hooks.useCommonMDMS(stateCode, "commonUiConfig", ["UserProfileValidationConfig"]);
+
+  useEffect(() => {
+    if(mdmsValidationData && mdmsValidationData?.UserProfileValidationConfig?.[0]){
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)